packages/algob/src/lib/status.ts

```diff
@@ -14,7 +14,7 @@ export async function balanceOf (
   }
 };
 
-export async function printAssets (deployer: AlgobDeployerImpl, account: string): Promise<void> {
+export async function printAssets (deployer: DeployerDeployMode, account: string): Promise<void> {
```

Comment:
This can be any deployer, so you should use the interface here.

Reply:
ok